### DISCUSSION TITLE
how do i make style like magnifier

### DISCUSSION THREAD

**Discussion by JackyBorderCollie** [CluckCluckChicken/Magnifier#90] [Q&A]
hey i wanted to use the magnifier's style on my website it's gonna be called magnifier tutorials but i don't know how to get da epik css style can someone help me?

**Comment by davwat** [comment, marked as answer]
it uses a framework called [tailwindcss](https://tailwindcss.com/) with [this config file](https://github.com/CluckCluckChicken/Magnifier/blob/main/Spyglass/tailwind.config.js). if you follow the instructions on the tailwindcss website for setting it up, then once tailwind.config.js gets generated, just replace it with the one i linked.

**Reply by JackyBorderCollie** [reply]
thx

**Comment by davwat** [comment]
also, a button looks like this:
<button @onclick="@onClick" class="transition duration-300 bg-primary hover:bg-primaryHover active:outline-none focus:outline-none text-black font-dosis text-4xl rounded-2xl p-3">@ChildContent</button>
you can see components and stuff [here](https://github.com/CluckCluckChicken/Magnifier/tree/main/Spyglass/Shared)
